Client/game_sa/CSettingsSA.cpp

```diff
@@ -590,6 +591,36 @@ void CSettingsSA::GetVehiclesLODDistance(float& fVehiclesLODDistance, float& fTr
     fTrainsPlanesLODDistance = ms_fTrainPlaneLODDistance;
 }
 
+////////////////////////////////////////////////
+//
+// Peds LOD draw distance
+//
+////////////////////////////////////////////////
+ 
+void CSettingsSA::SetPedsLODDistance(float fPedsLODDistance)
+{
+    ms_fPedsLODDistance = fPedsLODDistance;
+}
+ 
+void CSettingsSA::GetPedsLODDistance(float& fPedsLODDistance)
+{
+    fPedsLODDistance = ms_fPedsLODDistance;
+}

```

Comment:
GetPedsLODDistance should return instead of taking a parameter by reference.
`float CSettingsSA::GetPedsLODDistance()
{
    return ms_fPedsLODDistance;
}`

Reply:
Then `CSettingsSA::GetVehiclesLODDistance` would need a quick refacto after that PR for code consistency.